src/Symfony/Component/Messenger/Worker.php

```diff
@@ -170,6 +170,10 @@ private function handleMessage(Envelope $envelope, ReceiverInterface $receiver,
 
     public function stop(): void
     {
+        if (null !== $this->logger) {
+            $this->logger->info('Stopping worker.');
```

Comment:
@ruudk with https://github.com/symfony/symfony/pull/42335 i just found, maybe we can use it
cc @okwinza and related to more recent comment https://github.com/symfony/symfony/pull/42723#issuecomment-909263669
thank you

Reply:
If #42335 gets merged before this one does, it would be easier to simply rebase and update this PR instead of creating another one.

But it should be fine either way. :)